README.md

```diff
@@ -52,6 +52,7 @@ __AWESOME__ apps counter: __135__ 🎉
 - [E-book Reader](#e-book-reader)
 - [Egg Timer](#egg-timer)
 - [Emulation](#emulation)
+- [Email clients](#email-clients)
```

Comment:
1. When I first started this list I used plurality, however I prefer simplicity and singularity. That is just my opinion.
2. What is more "official" `Email` or `E-mail`?
> [Electronic mail (email or e-mail)](https://wikiless.org/wiki/Email?lang=en)

What do you think?
3. I think it is better to use just `Email` instead of `Email clients`

Reply:
Ok will to `# Email`